### PR TITLE
RPP Audio Support HIP - Spectrogram

### PR DESCRIPTION
**Pull request by r-abishek**
- Updates version to RPP 1.9.3
- Adds RPP audio support for Spectrogram on HIP backend.
- Adds relevant qa tests and performance tests.

### CODE REVIEW

**Comment by kiritigowda** [comment]
@r-abishek -- merge conflicts

File: src/modules/hip/kernel/spectrogram.hpp [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    }
+    else
+    {
+        windowFn = windowFunction;
+    }
+
+    // copy the hanning window values to hip memory
+    Rpp32f *d_windowFn = handle.GetInitHandle()->mem.mgpu.scratchBufferHip.floatmem;
+    CHECK_RETURN_STATUS(hipMemcpyAsync(d_windowFn, windowFn, windowLength * sizeof(Rpp32f), hipMemcpyHostToDevice, handle.GetStream()));
+    CHECK_RETURN_STATUS(hipStreamSynchronize(handle.GetStream()));
```
Comment (AryanSalmanpour) [inline src/modules/hip/kernel/spectrogram.hpp:185]: Please remove `CHECK_RETURN_STATUS(hipStreamSynchronize(handle.GetStream())));` from here and line 198 and move it to after the last hipLaunchKernelGGL call.

**Review by AryanSalmanpour — CHANGES_REQUESTED** [review]

**Comment by r-abishek** [comment]
@AryanSalmanpour This PR addresses verbal comments from the code review now

**Review by AryanSalmanpour — APPROVED** [review]

**Comment by r-abishek** [comment]
All Jenkins CI passing. Waiting on Azure - https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=10403&view=results

**Review by randyh62 — APPROVED** [review]
Looks good to me.

**Pull request merged**